src/ambianic/pipeline/ai/fall_detect.py

```diff
@@ -169,47 +195,33 @@ def fall_detect(self, image=None):
 
                 if not self._prev_vals or lapse > self.max_time_between_frames:
                     log.info("No recent pose to compare to. Will save this frame pose for subsequent comparison.")
+                elif not self.is_body_line_motion_downward(left_angle_with_yaxis, rigth_angle_with_yaxis):
+                    log.info("The body-line angle with vertical axis is decreasing from the previous frame.")
                 else:
-                    left_angle = 0
-                    left_body_line_detected = len(pose_vals_list[0]) == 2
-                    if left_body_line_detected and (self._prev_vals[0] is not None):
-                        try:
-                            temp_left_point = [[self._prev_vals[0][0], self._prev_vals[0][1]], [pose_vals_list[0][0], pose_vals_list[0][1]]]
-                            left_angle = self.calculate_angle(temp_left_point)
-                            log.info("Left shoulder-hip angle: %r", left_angle)
-                        except IndexError as e:
-                            log.exception("Error while calculating angle: %r", e)
-                    right_angle = 0
-                    right_body_line_detected = len(pose_vals_list[1]) == 2
-                    if right_body_line_detected and (self._prev_vals[1] is not None):
-                        try:
-                            temp_right_point = [[self._prev_vals[1][0], self._prev_vals[1][1]], [pose_vals_list[1][0], pose_vals_list[1][1]]]
-                            right_angle = self.calculate_angle(temp_right_point)
-                            log.info("Right shoulder-hip angle: %r", left_angle)
-                        except IndexError as e:
-                            log.exception("Error while calculating angle: %r", e)
+                    temp_left_point = [self._prev_vals[0], pose_vals_list[0]]
```

Comment:
Is `temp_left_point` the correct choice of variable name? Should it be `temp_left_vector` since it contains two points for the body shoulder-hip vector?

Reply:
changed...

Reply:
I don't see a commit.